resources/assets/js/components/topic.vue

```diff
@@ -0,0 +1,157 @@
+                        <svg v-show="! topic.userVotedFor" class="icon icon-arrow-up" transition="expand"><use xlink:href="#icon-arrow-up"></use></svg>
+                        <svg v-show="topic.userVotedFor" class="icon icon-checkmark" transition="expand"><use xlink:href="#icon-checkmark"></use></svg>
+                    </div>
+                </a><br>
+                <div class="vote-button__count">
+                    {{ topic.votes }}
+                </div>
+            </div>
+            <div class="col-xs-9 col-sm-10">
+                <div class="panel panel-default topic topic--in-list">
+                    <div class="panel-heading">
+                        <h3 class="topic__title">
+                            <a v-link="{ path: '/topics/' + topic.id }">{{ topic.title }}</a>
+                        </h3>
+                    </div>
+                    <div class="panel-body">
+                        {{ topic.description }}
+                    </div>
+                </div>
+            </div>
+            <div class="col-xs-9 col-sm-10" v-if="! loadingComments">
+                <h3>Comments</h3>
+
+                <div class="media" v-for="comment in comments">
+                    <div class="media-left">
+                        <img class="media-object" :src="comment.user.avatar" alt="{{ comment.user.name }}">
+                    </div>
+                    
+                    <div class="media-body">
+                        <h4 class="media-heading">{{ comment.user.name }} said, {{ comment.time_ago }}</h4>
+                        <p>{{ comment.body }}</p>
+                    </div>
+                </div>
+
+                <p class="text-muted" v-if="comments.length < 1">
+                    There are no comments yet. Why not post the first?
+                </p>
+
+                <br><br><form method="post" @submit.prevent="postComment">
+                    <div class="form-group">
+                        <textarea v-model="newComment.body" rows="3" class="form-control" placeholder="Enter a comment"></textarea>
+                    </div>
+
+                    <input type="submit" class="btn btn-primary" value="Post Comment">
+                </form>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        props: {
+            topics: {
+                sync: true
+            }
+        },
+        data: function () {
+            return {
+                topic: {},
+                comments: [],
+                loadingComments: true,
+                newComment: {}
+            };
+        },
+        created: function() {
+            var vm = this;
+
+            this.$http.get('topics', function(data, status, request) {
+                this.topic = _.find(this.topics, function (topic) {
+                    return topic.id == vm.$route.params.topic_id;
+                });
+
+                this.storeComments();
```

Comment:
Yeah, I did it that way to keep it consistent with the other components. I'm familiar with ES6 & promises though, I'll refactor all the components to use them in a separate PR :)